roles/bwc_rbac/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+st2_config_file_path: /etc/st2/st2.conf
```

Comment:
I think we can hardcode it without any big worries. `st2.conf` path is pretty much a constant everywhere, bundled in packaging.